src/types/parser/randomValue.ts

```diff
@@ -82,6 +82,10 @@ export function randomTreeOfObjectLiteral(
   for (const type_key_value of typeTree.children) {
     const [type_key_tree, type_value_tree] = type_key_value.children
 
+    if (type_value_tree.type === TreeNodeType.Invalid) {
+      continue;
```

Comment:
Consider configuring the prettier extension on your editor or run `npm run format` to remove the `;`.